### PR TITLE
BRMS-37 | Add move and trash icons

### PR DESCRIPTION
**Pull request by renanpvaz**
#### :thinking: What?
Two missing icons

#### :man_shrugging: Why?
Because I need them and they were in the assets already

#### :pushpin: Jira Issue
[BRMS-37](https://paacklogistics.atlassian.net/browse/BRMS-37)

### CODE REVIEW

File: src/UI/Icon.elm [hunk 1]
```diff
@@ -443,6 +445,26 @@ remove hint =
     Icon (Properties hint Remove) defaultOptions
 
 
+{-| A trash can with an "x" on it.
+
+    Icon.trash "Delete item"
+
+-}
+trash : String -> Icon
```
Comment (PedroHLC) [inline src/UI/Icon.elm:453]: Icons are named after their action, so we don't misuse it to designate two different actions.
In this case, I prefer it named `Delete`.

**Review by PedroHLC — CHANGES_REQUESTED** [review]

**Review by PedroHLC — APPROVED** [review]
:LGTM:

**Review by jouderianjr — APPROVED** [review]

File: src/UI/Icon.elm [hunk 2]
```diff
@@ -443,6 +445,26 @@ remove hint =
     Icon (Properties hint Remove) defaultOptions
 
 
+{-| A trash can with an "x" on it.
+
+    Icon.delete "Delete item"
+
+-}
+delete : String -> Icon
```
Comment (jouderianjr) [inline src/UI/Icon.elm:453]: check this Item because I cant see it in the showcase

**Review by jouderianjr — CHANGES_REQUESTED** [review]

File: src/UI/Icon.elm (hunk 2 above)
Comment (renanpvaz) [reply src/UI/Icon.elm:453]: oops find n replace gone wrong

**Review by jouderianjr — APPROVED** [review]

**Pull request merged**